### ISSUE TITLE
Refactor examples

### ISSUE THREAD

**Issue by Sytten** [obmarg/graphql-ws-client#37]
I think we should refactor examples to accommodate for WASM and graphql-client.
Probably using different packages.
What do you think?

**Comment by obmarg** [comment]
Yeah, not against refactoring the examples now there's support for just cynic.  What did you have in mind re: different packages?

**Comment by obmarg** [comment]
If you just mean separate packages for cynic examples, graphql-client examples etc. then yeah - sounds good to me

**Comment by HuakunShen** [comment]
Is there examples for `graphql-client`? That would be very helpful.

**Comment by obmarg** [comment]
@HuakunShen - I'm afraid I've never used this library with `graphql-client`.  

@Sytten - don't suppose you've got any example code you can share?

**Comment by obmarg** [comment]
@HuakunShen you could probably borrow some code from [the railway CLI](https://github.com/railwayapp/cli/blob/8ac14ef4a9161d2b7bd28ffbb14a329343d72bd3/src/subscription.rs) where they're using this library with `graphql_client`

**Comment by HuakunShen** [comment]
@obmarg Thank you!

In the README, it says there is integration with [graphql-client](https://github.com/graphql-rust/graphql-client): features = ["client-graphql-client"]

What does this mean? I thought `graphql-client` is supported.

I will take a look at railway first.

**Comment by obmarg** [comment]
> @obmarg Thank you!
> 
> In the README, it says there is integration with [graphql-client](https://github.com/graphql-rust/graphql-client): features = ["client-graphql-client"]
> 
> What does this mean? I thought `graphql-client` is supported.

It means that you need to enable the `client-graphql-client` feature flag to get `graphql-client` support [like the railway people have done here](https://github.com/railwayapp/cli/blob/master/Cargo.toml#L61).  We have support for multiple graphql clients, and control which are enabled via feature flags.

**Comment by obmarg** [comment]
Migrated to https://codeberg.org/obmarg/graphql-ws-client/issues/37
